src/HTML5Backend.js

```diff
@@ -34,25 +34,26 @@ export default class HTML5Backend {
     this.endDragNativeItem = this.endDragNativeItem.bind(this);
   }
 
-  setup() {
-    if (typeof window === 'undefined') {
+  setup(target = window) {
```

Comment:
When it finalizes, it will look something like:
```
render() {
  <DragDropContext backend={HTML5Backend} context={{window: iframeWindowInstance}}>
   // your app
  </DragDrapContext>
}
```